packages/blockchain-wallet-v4-frontend/src/components/Announcements/Service/index.js

```diff
@@ -24,15 +21,31 @@ class ServiceAnnouncement extends React.PureComponent {
     const { alertArea, data } = this.props
     return data.cata({
       Success: val => {
-        return val.showAnnounce ||
-          path(['announcements', alertArea, 'hideType'], val) === 'collapse' ? (
-          <Announcement
-            announcement={val.announcements[alertArea]}
-            language={val.language}
-            collapsed={val.collapsed}
-            handleDismiss={this.handleDismiss}
-            toggleCollapse={this.toggleCollapse}
-          />
+        return val.showTestnetWarning || val.showAnnounce ? (
+          <React.Fragment>
+            {val.showTestnetWarning && (
```

Comment:
Why can't testnet warning come from wallet-options-v4.json for only testnet?

Reply:
I see that makes sense, however for the ~1year we had testnet running we only showed this one banner. Announcement banners in general should be used sparingly and only in cases people could lose funds. Considering testnet is all fake funds I think one main public/wallet banner that says these funds are not real should suffice.